website_sale_recently_viewed_products/README.rst

```diff
@@ -0,0 +1,50 @@
+========================
```

Comment:
Why remove that?
I use it to make the module name a title, everything else is a "subsection" of that. It doesn't really make sense IMO that the module name should be a section at the same level of "Installation", "Usage" and so on.